flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/reconciler/JobReconciler.java

```diff
@@ -138,17 +134,20 @@ private void restoreFromLastSavepoint(FlinkDeployment flinkApp, Configuration ef
         deployFlinkJob(flinkApp, effectiveConfig, Optional.of(savepointLocation));
     }
 
-    private Optional<String> suspendJob(FlinkDeployment flinkApp, Configuration effectiveConfig)
-            throws Exception {
-        LOG.info("Suspending {}", flinkApp.getMetadata().getName());
-        return cancelJob(flinkApp, UpgradeMode.SAVEPOINT, effectiveConfig);
-    }
-
-    private Optional<String> cancelJob(FlinkDeployment flinkApp, Configuration effectiveConfig)
-            throws Exception {
-        LOG.info("Cancelling {}", flinkApp.getMetadata().getName());
-        UpgradeMode upgradeMode = flinkApp.getSpec().getJob().getUpgradeMode();
-        return cancelJob(flinkApp, upgradeMode, effectiveConfig);
+    private void printCancelLogs(UpgradeMode upgradeMode, String name) {
```

Comment:
Can this log integrate with the  `org.apache.flink.kubernetes.operator.service.FlinkService#cancelJob` ?

Reply:
If we failed to execute `getClusterClient()` in `FlinkService#cancelJob`, the logs will not show up. So I suggest to keep it here.

Reply:
Get it